mempool/src/core_mempool/mempool.rs

```diff
@@ -268,16 +293,36 @@ impl Mempool {
             aptos_infallible::duration_since_epoch_at(&now) + self.system_transaction_timeout;
 
         let txn_info = MempoolTransaction::new(
-            txn,
+            txn.clone(),
             expiration_time,
             ranking_score,
             timeline_state,
             db_sequence_number,
             now,
             client_submitted,
+            priority.clone(),
         );
 
+        let submitted_by_label = txn_info.insertion_info.submitted_by_label();
         let status = self.transactions.insert(txn_info);
+        let now = now
```

Comment:
Use `aptos_infallible::duration_since_epoch().as_millis()` to be consistent with the rest of the file?

Reply:
Done.